libs/pika/async_mpi/tests/unit/mpi_async_storage.cpp

```diff
@@ -39,12 +39,11 @@
 
 // a debug level of zero disables messages with a priority>0
 // a debug level of N shows messages with priority<N
-constexpr int debug_level = 0;
+constexpr int debug_level = 9;
```

Comment:
This means debugging is enabled, right? Should this stay or is it just temporary?

Reply:
0 means messages are squashed. 9 means all messages from 0-9 are shown. Mostly I set important messages to low numbers, and less important to high numbers, then set debug level to around 4 or 5 - unless I need to see everything. A value of 0 turn stuff off, unless of course you instantiate a debug object of level 0, but I only use 0 for critical messages that are not really debug only. (c.f. warning, debug, trace, type levels but numerical)